sdk/textanalytics/ai-text-analytics/test/public/pipelineOptions.spec.ts

```diff
@@ -2,23 +2,21 @@
 // Licensed under the MIT license.
 
 import { assert } from "chai";
+import * as dotenv from "dotenv";
 
-import {
-  TextAnalyticsClient,
-  DetectLanguageResultArray,
-  DetectLanguageSuccessResult,
-  AzureKeyCredential
-} from "../../src";
-import { testEnv } from "../utils/recordedClient";
+import { DetectLanguageResultArray, DetectLanguageSuccessResult } from "../../src";
 
-import { WebResource, HttpOperationResponse, HttpHeaders } from "@azure/core-http";
+import { WebResource, HttpOperationResponse, HttpHeaders, isNode } from "@azure/core-http";
+import { createClient } from "../utils/recordedClient";
 
-describe("TextAnalyticsClient Custom PipelineOptions", function() {
-  const credential = new AzureKeyCredential(testEnv.TEXT_ANALYTICS_API_KEY);
+if (isNode) {
+  dotenv.config();
```

Comment:
This seems like it would introduce bundler warnings through dotenv's dependency on fs-extra. If not, then ignore this.

Reply:
`rush build -t .` did not produce warnings if this is what you meant.
![build](https://user-images.githubusercontent.com/6074665/102911676-5f2d3900-444a-11eb-8985-893a3ed2058c.PNG)

Reply:
You won't see bundler warnings in the rush build output because all rollup output is redirected (otherwise every package would warn from `rush build`. Does `rushx build` in text-analytics have a warning?

Reply:
it does, hmmmm:
```
$ rushx build
Found configuration in /home/dealmaha/textCIFail/rush.json


Rush Multi-Project Build Tool 5.28.0 - https://rushjs.io
Node.js version is 14.13.1 (pre-LTS)

Found configuration in /home/dealmaha/textCIFail/rush.json

Executing: "tsc -p . && rollup -c 2>&1 && api-extractor run --local"


./dist-esm/src/index.js → dist/index.js...
created dist/index.js in 526ms

dist-esm/test/**/*.spec.js, dist-esm/test/**/node/** → dist-test/index.browser.js...
(!) Missing shims for Node.js built-ins
Creating a browser bundle that depends on 'path'. You might need to include https://www.npmjs.com/package/rollup-plugin-node-builtins
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
fs (imported by ../../../common/temp/node_modules/.pnpm/dotenv@8.2.0/node_modules/dotenv/lib/main.js, fs?commonjs-external)
path (imported by ../../../common/temp/node_modules/.pnpm/dotenv@8.2.0/node_modules/dotenv/lib/main.js, path?commonjs-external)
(!) Missing global variable names
Use output.globals to specify browser global variable names corresponding to external modules
fs (guessing 'fs')
path (guessing 'path')
created dist-test/index.browser.js in 6.5s

api-extractor 7.7.11  - https://api-extractor.com/

Using configuration from /home/dealmaha/textCIFail/sdk/textanalytics/ai-text-analytics/api-extractor.json


API Extractor completed successfully
```

Reply:
Fixed inhttps://github.com/Azure/azure-sdk-for-js/pull/12991/commits/a891925c25663a1e30df91057705f33ff8e8382e.